index.js

```diff
@@ -1,9 +1,18 @@
 module.exports = {
     env: {
         browser: true,
-        node: true,
-        es6: true
+        node: true
     },
+    extends: [
+        './possible-errors.js',
+        './best-practices.js',
+        './strict-mode.js',
+        './variables.js',
+        './node-js-and-common-js.js',
+        './stylistic-issues.js',
+        './ecma-script-6.js'
+    ],
+    rules: {},
     globals: {
```

Comment:
I think globals setup should go with environments, e.g. we have different globals needed for a test environment, for a browser and a node environment.